site/en/configure/coverage.md

```diff
@@ -122,7 +122,7 @@ remote execution, as well, including JUnit.
 
 ### Python
 
-See the [`rules_python` coverage docs](https://github.com/bazelbuild/rules_python/blob/main/docs/sphinx/coverage.md)
+See the [`rules_python` coverage docs](https://github.com/bazelbuild/rules_python/blob/main/docs/coverage.md)
```

Comment:
Thanks! Could you instead link to the generated RTD docs that you can find through the main repo readme? I think those links maybe more stable than the source code.

Reply:
Sure thing! Updated in https://github.com/bazelbuild/bazel/pull/25150/commits/b2b688eb08a16d168506407a8a2f1ae9a192f2f5.